addons/mail/static/src/components/attachment_link_preview/attachment_link_preview.js

```diff
@@ -0,0 +1,35 @@
+/** @odoo-module **/
+
+import { useComponentToModel } from '@mail/component_hooks/use_component_to_model/use_component_to_model';
+import { registerMessagingComponent } from '@mail/utils/messaging_component';
+
+const { Component } = owl;
+
+class AttachmentLinkPreview extends Component {
+
+    setup() {
+        super.setup();
+        useComponentToModel({ fieldName: 'component', modelName: 'AttachmentLinkPreviewView' });
+    }
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * @returns {AttachmentLinkPreviewView}
+     */
+    get attachmentLinkPreviewView() {
+        return this.messaging && this.messaging.models['AttachmentLinkPreviewView'].get(this.props.localId);
+    }
+
+}
+
+Object.assign(AttachmentLinkPreview, {
+    props: {
+        localId: String,
+    },
```

Comment:
Nit-picking but this could be a one-liner.
When all view model are introduced (and we still have to define components), all of them will just have `localId` as props.

Reply:
I find it less readable, Object should be multi line, but ok since it does kill 2 LOC...